[0m11:57:44.191393 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8144f6aa00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8143618e80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8143479640>]}


============================== 11:57:44.201987 | 24a84a01-4cc7-4eb6-8137-d7573d8d726f ==============================
[0m11:57:44.201987 [info ] [MainThread]: Running with dbt=1.8.0
[0m11:57:44.203676 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/achraf/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init rt-analytics', 'send_anonymous_usage_stats': 'True'}
[0m11:57:44.215193 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/achraf/.dbt')}
[0m11:57:44.216444 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:57:59.931003 [debug] [MainThread]: Starter project path: /home/achraf/.local/lib/python3.8/site-packages/dbt/include/starter_project
[0m11:58:00.170182 [info ] [MainThread]: 
Your new dbt project "rt_analytics" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:58:00.171411 [info ] [MainThread]: Setting up your profile.
[0m11:58:00.174101 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m11:58:00.177332 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 16.178303, "process_user_time": 1.480823, "process_kernel_time": 0.463386, "process_mem_max_rss": "88952", "process_in_blocks": "5664", "command_success": false, "process_out_blocks": "0"}
[0m11:58:00.178649 [debug] [MainThread]: Command `dbt init` failed at 11:58:00.178472 after 16.18 seconds
[0m11:58:00.179697 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8144f6aa00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8143412fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8143412130>]}
[0m11:58:00.180791 [debug] [MainThread]: Flushing usage events
